quran/models.py

```diff
@@ -1,19 +1,97 @@
 '''Models describing abstractions used at Quran app'''
-from django.db import migrations, models
+import os
+from django.db import models
+from django.template.defaultfilters import slugify
+from .fields import SegmentsField
 
 
 class Surah(models.Model):
-    '''General information about Surahs'''
+    '''Model representing a surah from the Qur’an.'''
     number = models.PositiveIntegerField(primary_key=True)
     name = models.CharField(max_length=20)
     total_ayahs = models.PositiveIntegerField()
 
+    def __str__(self):
+        return f'{self.name}: ({self.number})'
+
 
 class Ayah(models.Model):
-    '''General information about Ayah'''
+    '''Model representing an ayah from the Qur’an.'''
     number = models.PositiveIntegerField()
     text = models.TextField()
     surah = models.ForeignKey(
         Surah,
         on_delete=models.CASCADE,
         related_name='ayahs')
+
+    def __str__(self):
+        return f'{self.surah.name}: ({self.surah.number}:{self.number})'
+
+
+class Reciter(models.Model):
+    '''Model representing a reciter'''
+    name = models.CharField(max_length=100)
+    bitrate = models.PositiveIntegerField(
+        blank=True,
+        null=True,
+        help_text="Bitrate of an audio file")
+    style = models.CharField(
+        max_length=20, blank=True,
+        help_text="Qur'an reading style")
+    slug = models.SlugField(
+        help_text="Short label for name, \
+        containing only letters and hyphens. \
+        It's filled automatically during saving.")
+
+    def save(self, *args, **kwargs):
+        # creating slug from name
+        self.slug = slugify(self.name)
+        super(Reciter, self).save(*args, **kwargs)
+
+    def __str__(self):
```

Comment:
This is too complicated. Rewrite it in an easier way. Something like: `Name (params)` and if some of the params is absent, put `None` there. Example: `Mishari (style=fast, bitrate=None)`